# Reproducibility of Papers from Various Scientific Fields

# Introduction

# Methods

# Discussion

## Human Genetics In Disease Identification (Alina Tan)

## Medical Imaging and Segmentation (Megan Ng)

## Iron Deficiency and Blood Donation (Matthew Schinwald)

## Bacterial Genomics in Health Settings (Monica Cella)

##  Neuroplasticity and Perceptual Learning (Ashim Pandey)

## Cortex-wide Brain Imaging and Neural Dynamics (Rian Fritz D. Jalandoni)

## Single Molecule Imaging of DNA Interacting Proteins (Lydia Hodgins)
5 papers related to single molecule tracking and analysis of the dynamics of DNA interacting proteins were evaluated as they relate to the research topic studied by contributor LH. These papers fall into the scientific fields of fundamental biology and biophysics. They range in publication years from 2019 to 2024 and represent the publication journals: Journal of Physical Chemistry, Nucleic Acids Research, Journal of Physics D: Applied Physics, eLife, and Molecular Cell. Overall, it is found that these publications perform well in the description of methods, sample population, and tools categories, but perform poorly in both the data and tool availability categories. When evaluating my personal impression of the reproducibility of these papers I found that data and tool availability had the biggest impact on my decision, resulting in most papers receiving a low score. 

Mine-Hattab et al. (2021) received the highest reproducibility score (4) and was the only paper that scored 3/3 in data availability. This study provided all raw data through a zenodo data base. Both the description of the methods and the sample populations are clear, detailed, and well organized. However, the description of the tools used in this study lacked detail such as microscope and software specifications. The study evaluated poorest in the tool availability category as all analysis programs were custom-built and not openly shared. This observation resulted in an overall reproducibility score of 4/5 as the data is available, however, analysis depends on either cooperation from the authors to share their code or use of another software chosen or developed based on the description of analysis methods. Either of which introduce uncertainty regarding the execution of reproducing the results. 

El Sayyed et al. (2024) received the next highest reproducibility score (3). This study had a very clear and detailed materials and methods section, scoring 3 across the board for description of methods, data and tools. Where this study performs poorly is the availability of both the data and tools. Only a portion of the raw data is openly accessible, which does not include the data which most significantly contributes to the conclusions. However, a statement is provided claiming, “any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request” (El Sayyed et al., 2024). With respect to the tools, links are provided for any open source code and software that was used, but the software used for the core analysis is declared to be custom and is not openly shared. The limited access to both the data and analysis tools is why this paper receives an overall score of 3/5 as it is possible the results could be reproduced but this is highly dependent on the authors complying to the claim that additional data and software will be shared.

Fan et al. (2023) and Bettridge et al. (2023) both received 2/5 for their overall reproducibility scores. For Fan et al. (2023) the study performed well with respect to the description of methods and data but performed poorly when describing the tools as import specifications regarding the microscope were omitted and few to no parameters were provided for both image acquisition and analysis. Additionally, no data or custom code are openly available. Only links to open-source software and a note stating data will be provided upon request are provided. For Bettridge et al. (2023) the study provided clear and detailed descriptions of the methods, data and tools. Again for this study, no data or custom analysis code is openly accessible. No statement is provided regarding the authors’ position on sharing the data, however, it is noted that some custom code is available upon request. The heavy reliance on compliance from the authors’ to share both data and analysis code to reproduce these studies is why both received a overall score of 2/5 as it is uncertain whether compliance would be experienced. 

Lastly, Banaz et al. (2019) received the lowest reproducibility score of 1/5 and performed very poorly across the board. The study lacked a materials and methods section and instead reports poorly detailed methods throughout the main text. Additionally, specific details regarding the population sample, repeats, microscope specifications, and utilized software or code are missing. No data is openly available and no acknowledgement regarding a willingnes to share the data is provided. With respect to the software tools, more information is necessary from the authors to identify what tools were used and how to access them.

## Machine learning for biological insights from sequence modelling  (César Miguel Valdez Córdova)

## Biomedical Engineering Methods for drug delivery and microfluidics (Ioan Duchastel)

## Pathophysiological Modelling for Optic Nerve Injury (Ryan Huang)

## Neural Signal Analysis of Anesthesia-Associated Brain Dynamics Change (Ryan Huang)


# Conclusion

# References

Afrashteh, N., Inayat, S., Mohsenvand, M., & Mohajerani, M. H. (2017). Optical-flow analysis toolbox for characterization of spatiotemporal dynamics in mesoscale optical imaging of brain activity. NeuroImage, 153, 58–74. https://doi.org/10.1016/j.neuroimage.2017.03.034

Awada, A., Bakhtiari, S., & Pack, C. C. (2021). Visual perceptual learning generalizes to untrained effectors. Journal of Vision, 21(3), 10. https://doi.org/10.1167/jov.21.3.10

Banaz, N., Mäkelä, J., & Uphoff, S. (2018). Choosing the right label for single-molecule tracking in live bacteria: Side-by-side comparison of photoactivatable fluorescent protein and Halo tag dyes. Journal of Physics D: Applied Physics, 52(6), 064002. https://doi.org/10.1088/1361-6463/aaf255

Bettridge, K., Harris, F. E., Yehya, N., & Xiao, J. (2023). RNAP Promoter Search and Transcription Kinetics in Live E. coli Cells. The Journal of Physical Chemistry B, 127(17), 3816–3828. https://doi.org/10.1021/acs.jpcb.2c09142

Boyle, E. A., Li, Y. I., & Pritchard, J. K. (2017). An Expanded View of Complex Traits: From Polygenic to Omnigenic. Cell, 169(7), 1177–1186. https://doi.org/10.1016/j.cell.2017.05.038

Bryant, B. J., Yau, Y. Y., Arceo, S. M., Daniel-Johnson, J., Hopkins, J. A., & Leitman, S. F. (2012). Iron Replacement Therapy in the Routine Management of Blood Donors. Transfusion, 52(7), 1566–1575. https://doi.org/10.1111/j.1537-2995.2011.03488.x

Calì, C., Agus, M., Kare, K., Boges, D. J., Lehväslaiho, H., Hadwiger, M., & Magistretti, P. J. (2019). 3D cellular reconstruction of cortical glia and parenchymal morphometric analysis from Serial Block-Face Electron Microscopy of juvenile rat. Progress in Neurobiology, 183, 101696. https://doi.org/10.1016/j.pneurobio.2019.101696

Chen, Y., Butler-Laporte, G., Liang, K. Y. H., Ilboudo, Y., Yasmeen, S., Sasako, T., Langenberg, C., Greenwood, C. M. T., & Richards, J. B. (2024). The performance of AlphaMissense to identify genes causing disease (p. 2024.03.05.24303647). medRxiv. https://doi.org/10.1101/2024.03.05.24303647

Chng, K. R., Li, C., Bertrand, D., Ng, A. H. Q., Kwah, J. S., Low, H. M., Tong, C., Natrajan, M., Zhang, M. H., Xu, L., Ko, K. K. K., Ho, E. X. P., Av-Shalom, T. V., Teo, J. W. P., Khor, C. C., Chen, S. L., Mason, C. E., Ng, O. T., Marimuthu, K., … Nagarajan, N. (2020). Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment. Nature Medicine, 26(6), 941–951. https://doi.org/10.1038/s41591-020-0894-4

Clapis, J. R., Fan, M. J., & Kovarik, M. L. (2021). Supported bilayer membranes for reducing cell adhesion in microfluidic devices. Analytical Methods, 13(12), 1535–1540. https://doi.org/10.1039/D0AY01992E

Couto, J., Musall, S., Sun, X. R., Khanal, A., Gluf, S., Saxena, S., Kinsella, I., Abe, T., Cunningham, J. P., Paninski, L., & Churchland, A. K. (2021). Chronic, cortex-wide imaging of specific cell populations during behavior. Nature Protocols, 16(7), 3241–3263. https://doi.org/10.1038/s41596-021-00527-z

DaSilva, L. F., Senan, S., Patel, Z. M., Reddy, A. J., Gabbita, S., Nussbaum, Z., Córdova, C. M. V., Wenteler, A., Weber, N., Tunjic, T. M., Khan, T. A., Li, Z., Smith, C., Bejan, M., Louis, L. K., Cornejo, P., Connell, W., Wong, E. S., Meuleman, W., & Pinello, L. (2024). DNA-Diffusion: Leveraging Generative Models for Controlling Chromatin Accessibility and Gene Expression via Synthetic Regulatory Elements (p. 2024.02.01.578352). bioRxiv. https://doi.org/10.1101/2024.02.01.578352

de Kort, W., Prinsze, F., Nuboer, G., Twisk, J., & Merz, E.-M. (2019). Deferral rate variability in blood donor eligibility assessment. Transfusion, 59(1), 242–249. https://doi.org/10.1111/trf.14984

Fan, J., El Sayyed, H., Pambos, O. J., Stracy, M., Kyropoulos, J., & Kapanidis, A. N. (2023). RNA polymerase redistribution supports growth in E. coli strains with a minimal number of rRNA operons. Nucleic Acids Research, 51(15), 8085–8101. https://doi.org/10.1093/nar/gkad511

Fang, R. H., Hu, C.-M. J., Luk, B. T., Gao, W., Copp, J. A., Tai, Y., O’Connor, D. E., & Zhang, L. (2014). Cancer Cell Membrane-Coated Nanoparticles for Anticancer Vaccination and Drug Delivery. Nano Letters, 14(4), 2181–2188. https://doi.org/10.1021/nl500618u

Forgetta, V., Jiang, L., Vulpescu, N. A., Hogan, M. S., Chen, S., Morris, J. A., Grinek, S., Benner, C., Jang, D.-K., Hoang, Q., Burtt, N., Flannick, J. A., McCarthy, M. I., Fauman, E., Greenwood, C. M. T., Maurano, M. T., & Richards, J. B. (2022). An effector index to predict target genes at GWAS loci. Human Genetics, 141(8), 1431–1447. https://doi.org/10.1007/s00439-022-02434-z

Fuller, J. T., Barnes, S., Sadun, L. A., Ajmera, P., Alexandrova, A. N., & Sadun, A. A. (2023). Coenzyme Q10 trapping in mitochondrial complex I underlies Leber’s hereditary optic neuropathy. Proceedings of the National Academy of Sciences, 120(39), e2304884120. https://doi.org/10.1073/pnas.2304884120

Hirase, H., Akther, S., Wang, X., & Oe, Y. (2019). Glycogen distribution in mouse hippocampus. Journal of Neuroscience Research, 97(8), 923–932. https://doi.org/10.1002/jnr.24386

Huxlin, K. R., Martin, T., Kelly, K., Riley, M., Friedman, D. I., Burgin, W. S., & Hayhoe, M. (2009). Perceptual Relearning of Complex Visual Motion after V1 Damage in Humans. Journal of Neuroscience, 29(13), 3981–3991. https://doi.org/10.1523/JNEUROSCI.4882-08.2009

Karregat, J., Sweegers, M. G., Quee, F. A., Weekamp, H. H., Swinkels, D. W., Novotny, V. M. J., Zaaijer, H. L., & Van Den Hurk, K. (2022). Ferritin-guided iron supplementation in whole blood donors: Optimal dosage, donor response, return and efficacy (FORTE)—a randomised controlled trial protocol. BMJ Open, 12(3), e056316. https://doi.org/10.1136/bmjopen-2021-056316

Kent, A. G., Vill, A. C., Shi, Q., Satlin, M. J., & Brito, I. L. (2020). Widespread transfer of mobile antibiotic resistance genes within individual gut microbiomes revealed through bacterial Hi-C. Nature Communications, 11(1), 4379. https://doi.org/10.1038/s41467-020-18164-7

Krayenbuehl, P.-A., Battegay, E., Breymann, C., Furrer, J., & Schulthess, G. (2011). Intravenous iron for the treatment of fatigue in nonanemic, premenopausal women with low serum ferritin concentration. Blood, 118(12), 3222–3227. https://doi.org/10.1182/blood-2011-04-346304

Lal, A., Garfield, D., Biancalani, T., & Eraslan, G. (2024). regLM: Designing realistic regulatory DNA with autoregressive language models (p. 2024.02.14.580373). bioRxiv. https://doi.org/10.1101/2024.02.14.580373

Lennon, N. J., Kottyan, L. C., Kachulis, C., Abul-Husn, N. S., Arias, J., Belbin, G., Below, J. E., Berndt, S. I., Chung, W. K., Cimino, J. J., Clayton, E. W., Connolly, J. J., Crosslin, D. R., Dikilitas, O., Velez Edwards, D. R., Feng, Q., Fisher, M., Freimuth, R. R., Ge, T., … Kenny, E. E. (2024). Selection, optimization and validation of ten chronic disease polygenic risk scores for clinical implementation in diverse US populations. Nature Medicine, 30(2), 480–487. https://doi.org/10.1038/s41591-024-02796-z

Liang, Y., Song, C., Liu, M., Gong, P., Zhou, C., & Knöpfel, T. (2021). Cortex-Wide Dynamics of Intrinsic Electrical Activities: Propagating Waves and Their Interactions. Journal of Neuroscience, 41(16), 3665–3678. https://doi.org/10.1523/JNEUROSCI.0623-20.2021

Linden, N. J., Tabuena, D. R., Steinmetz, N. A., Moody, W. J., Brunton, S. L., & Brunton, B. W. (2021). Go with the FLOW: Visualizing spatiotemporal dynamics in optical widefield calcium imaging. Journal of The Royal Society Interface, 18(181), 20210523. https://doi.org/10.1098/rsif.2021.0523

Mantadakis, E., Panagopoulou, P., Kontekaki, E., Bezirgiannidou, Z., & Martinis, G. (2022). Iron Deficiency and Blood Donation: Links, Risks and Management. Journal of Blood Medicine, 13, 775–786. https://doi.org/10.2147/JBM.S375945

Marbouty, M., Cournac, A., Flot, J.-F., Marie-Nelly, H., Mozziconacci, J., & Koszul, R. (2014). Metagenomic chromosome conformation capture (meta3C) unveils the diversity of chromosome organization in microorganisms. eLife, 3, e03318. https://doi.org/10.7554/eLife.03318

Miné-Hattab, J., Heltberg, M., Villemeur, M., Guedj, C., Mora, T., Walczak, A. M., Dahan, M., & Taddei, A. (2021). Single molecule microscopy reveals key physical features of repair foci in living cells. eLife, 10, e60577. https://doi.org/10.7554/eLife.60577

Nguyen, E., Poli, M., Durrant, M. G., Thomas, A. W., Kang, B., Sullivan, J., Ng, M. Y., Lewis, A., Patel, A., Lou, A., Ermon, S., Baccus, S. A., Hernandez-Boussard, T., Ré, C., Hsu, P. D., & Hie, B. L. (2024). Sequence modeling and design from molecular to genome scale with Evo (p. 2024.02.27.582234). bioRxiv. https://doi.org/10.1101/2024.02.27.582234

Nguyen, E., Poli, M., Faizi, M., Thomas, A., Birch-Sykes, C., Wornow, M., Patel, A., Rabideau, C., Massaroli, S., Bengio, Y., Ermon, S., Baccus, S. A., & Ré, C. (2023). HyenaDNA: Long-Range Genomic Sequence Modeling at Single Nucleotide Resolution (arXiv:2306.15794). arXiv. https://doi.org/10.48550/arXiv.2306.15794

Ochoa, D., Karim, M., Ghoussaini, Maya, Hulcoop, D. G., McDonagh, E. M., & Dunham, I. (2022). Human genetics evidence supports two-thirds of the 2021 FDA-approved drugs. Nature Reviews Drug Discovery, 21(8), 551–551. https://doi.org/10.1038/d41573-022-00120-3

Pan, Y., Sullivan, D., Shreiber, D. I., & Pelegri, A. A. (2013). Finite Element Modeling of CNS White Matter Kinematics: Use of a 3D RVE to Determine Material Properties. Frontiers in Bioengineering and Biotechnology, 1. https://doi.org/10.3389/fbioe.2013.00019

Plourde, G., & Arseneau, F. (2017). Attenuation of high-frequency (30–200 Hz) thalamocortical EEG rhythms as correlate of anaesthetic action: Evidence from dexmedetomidine. British Journal of Anaesthesia, 119(6), 1150–1160. https://doi.org/10.1093/bja/aex329

Rao, L., He, Z., Meng, Q.-F., Zhou, Z., Bu, L.-L., Guo, S.-S., Liu, W., & Zhao, X.-Z. (2017). Effective cancer targeting and imaging using macrophage membrane-camouflaged upconversion nanoparticles. Journal of Biomedical Materials Research Part A, 105(2), 521–530. https://doi.org/10.1002/jbm.a.35927

Rohlfing, T. (2012). Image Similarity and Tissue Overlaps as Surrogates for Image Registration Accuracy: Widely Used but Unreliable. IEEE Transactions on Medical Imaging, 31(2), 153–163. https://doi.org/10.1109/TMI.2011.2163944

Salmon, C. K., Syed, T. A., Kacerovsky, J. B., Alivodej, N., Schober, A. L., Sloan, T. F. W., Pratte, M. T., Rosen, M. P., Green, M., Chirgwin-Dasgupta, A., Mehta, S., Jilani, A., Wang, Y., Vali, H., Mandato, C. A., Siddiqi, K., & Murai, K. K. (2023). Organizing principles of astrocytic nanoarchitecture in the mouse cerebral cortex. Current Biology, 33(5), 957-972.e5. https://doi.org/10.1016/j.cub.2023.01.043

Sayyed, H. E., Pambos, O. J., Stracy, M., Gottesman, M. E., & Kapanidis, A. N. (2024). Single-molecule tracking reveals the functional allocation, in vivo interactions, and spatial organization of universal transcription factor NusG. Molecular Cell, 84(5), 926-937.e4. https://doi.org/10.1016/j.molcel.2024.01.025

Serizay, J., Matthey-Doret, C., Bignaud, A., Baudry, L., & Koszul, R. (2024). Orchestrating chromosome conformation capture analysis with Bioconductor. Nature Communications, 15(1), 1072. https://doi.org/10.1038/s41467-024-44761-x

Stalder, T., Press, M. O., Sullivan, S., Liachko, I., & Top, E. M. (2019). Linking the resistome and plasmidome to the microbiome. The ISME Journal, 13(10), 2437–2446. https://doi.org/10.1038/s41396-019-0446-4

Szinte, M., Jonikaitis, D., Rolfs, M., Cavanagh, P., & Deubel, H. (2016). Presaccadic motion integration between current and future retinotopic locations of attended objects. Journal of Neurophysiology, 116(4), 1592–1602. https://doi.org/10.1152/jn.00171.2016

Tesser, R. A., Niendorf, E. R., & Levin, L. A. (2003). The morphology of an infarct in nonarteritic anterior ischemic optic neuropathy. Ophthalmology, 110(10), 2031–2035. https://doi.org/10.1016/S0161-6420(03)00804-2

Theodoris, C. V., Xiao, L., Chopra, A., Chaffin, M. D., Al Sayed, Z. R., Hill, M. C., Mantineo, H., Brydon, E. M., Zeng, Z., Liu, X. S., & Ellinor, P. T. (2023). Transfer learning enables predictions in network biology. Nature, 618(7965), 616–624. https://doi.org/10.1038/s41586-023-06139-9

Townsend, R. G., & Gong, P. (2018). Detection and analysis of spatiotemporal patterns in brain activity. PLoS Computational Biology, 14(12), e1006643. https://doi.org/10.1371/journal.pcbi.1006643

Willis, H. E., Ip, I. B., Watt, A., Campbell, J., Jbabdi, S., Clarke, W. T., Cavanaugh, M. R., Huxlin, K. R., Watkins, K. E., Tamietto, M., & Bridge, H. (2023). GABA and Glutamate in hMT+ Link to Individual Differences in Residual Visual Function After Occipital Stroke. Stroke, 54(9), 2286–2295. https://doi.org/10.1161/STROKEAHA.123.043269

Xie, G., Deschamps, A., Backman, S. B., Fiset, P., Chartrand, D., Dagher, A., & Plourde, G. (2011a). Critical involvement of the thalamus and precuneus during restoration of consciousness with physostigmine in humans during propofol anaesthesia: A positron emission tomography study. British Journal of Anaesthesia, 106(4), 548–557. https://doi.org/10.1093/bja/aeq415

Xie, G., Deschamps, A., Backman, S. B., Fiset, P., Chartrand, D., Dagher, A., & Plourde, G. (2011b). Critical involvement of the thalamus and precuneus during restoration of consciousness with physostigmine in humans during propofol anaesthesia: A positron emission tomography study. British Journal of Anaesthesia, 106(4), 548–557. https://doi.org/10.1093/bja/aeq415

Yang 杨菁艺, J., Saionz, E. L., Cavanaugh, M. R., Fahrenthold, B. K., Melnick, M. D., Tadin, D., Briggs, F., Carrasco, M., & Huxlin, K. R. (2024). Limited restoration of contrast sensitivity with training after V1 damage in humans. eNeuro, 11(3), ENEURO.0020-24.2024. https://doi.org/10.1523/ENEURO.0020-24.2024


